src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+            cursor: {}
+
+- Name: ExpressionReverseArray
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionReverseArray
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {reverseArray: { $reverseArray: $array}}}]
+            cursor: {}
+
+- Name: ExpressionIsArray
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionIsArray
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {isArray: { $isArray: $randomType}}}]
+            cursor: {}
+
+- Name: ExpressionSqrt
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionSqrt
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {sqrt: { $sqrt: $posInt}}}]
+            cursor: {}
+
+- Name: ExpressionSwitch
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionSwitch
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {switch: { $switch: {branches: [{case: {$gte: [$int, 0]}, then: "GTE 0"}, {case: {$lt: [$int, 0]}, then: "LT 0"}], default: "Out of bounds"}}}}]
+            cursor: {}
+
+- Name: ExpressionTestToLower
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionToLower
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {toLower: { $toLower: $string}}}]
```

Comment:
recommend using an all upper case string here, inverse for toUpper, use an all lower case string.

Reply:
Done.